templates/blocks/site_logo.twig

```diff
@@ -5,6 +5,7 @@
 		<img
 			src="{{ data_nav_bar.images }}gp-logo-{{ logo_color }}.svg"
 			alt="Greenpeace"
+			title="Greenpeace"
```

Comment:
I don't think we need a tooltip in the site logo, the author profile image, or even the article thumbnails on search and articles block (the other PR). 

According to the [original ticket](https://jira.greenpeace.org/browse/PLANET-1757) the intention was mostly to add tooltips on images being included on posts and pages so users can easily see credit on hover. I don't think we should expand the scope of the initial code.

Reply:
Yes got the point 👍, It was misunderstood by me as applicable over all images.

I will update both the PR accordingly 🙂

Reply:
Both PRs updated, please have a look.